src/sentry/tasks/sentry_apps.py

```diff
@@ -174,24 +184,23 @@ def _process_resource_change(action, sender, instance_id, retryer=None, *args, *
 
 
 @instrumented_task("sentry.tasks.process_resource_change", **TASK_OPTIONS)
-@retry()
+@retry(ignore=(IgnorableSentryAppError))
 def process_resource_change(action, sender, instance_id, *args, **kwargs):
     _process_resource_change(action, sender, instance_id, *args, **kwargs)
 
 
 @instrumented_task("sentry.tasks.process_resource_change_bound", bind=True, **TASK_OPTIONS)
-@retry()
+@retry(ignore=(IgnorableSentryAppError))
 def process_resource_change_bound(self, action, sender, instance_id, *args, **kwargs):
     _process_resource_change(action, sender, instance_id, retryer=self, *args, **kwargs)
 
 
 @instrumented_task(name="sentry.tasks.sentry_apps.installation_webhook", **TASK_OPTIONS)
-@retry(on=(RequestException,))
+@retry(on=(RequestException, ApiHostError, ApiTimeoutError), ignore=(IgnorableSentryAppError))
```

Comment:
@ceorourke I think it would be better to define all of the options at the top of the file like we do for `TASK_OPTIONS` so we don't have to keep repeating this code